mapie/regression/regression.py

```diff
@@ -734,48 +733,18 @@ def predict(
             if ensemble:
                 y_pred = aggregate_all(self.agg_function, y_pred_multi)
 
-        # compute distributions of lower and upper bounds
-        if self.conformity_score_function_.sym:
-            conformity_scores_low = -self.conformity_scores_
-            conformity_scores_up = self.conformity_scores_
-        else:
-            conformity_scores_low = self.conformity_scores_
-            conformity_scores_up = self.conformity_scores_
-            alpha_np = alpha_np / 2
-
-        lower_bounds = (
-            self.conformity_score_function_.get_estimation_distribution(
-                y_pred_multi_low, conformity_scores_low
+        # get desired confidence intervals according to alpha
```

Comment:
Not in the scope of this PR. cf comment for the new conformity_score file

Reply:
Thank you to notice that mistake.